s3/replication/manager/setup.cfg

```diff
@@ -48,4 +48,3 @@ where = src
 config =
     src/config/config.yaml
     src/config/logger_config.yaml
-    src/config/source_target_s3_config.yaml
```

Comment:
Good catch. We may need to add aws_s3.yaml and cortx_s3.yaml

Reply:
done.